kratos/python/add_other_utilities_to_python.cpp

```diff
@@ -792,6 +793,17 @@ void AddOtherUtilitiesToPython(pybind11::module &m)
         .def(py::init<Model&, Parameters>())
         .def("CalculateExtensionOperator", &ShiftedBoundaryMeshlessInterfaceUtility::CalculateExtensionOperator)
     ;
+
+    m.def_submodule("StringUtilities", "Free-floating utility functions for string manipulation.")
+        .def("ConvertCamelCaseToSnakeCase",
+             StringUtilities::ConvertCammelCaseToSnakeCase,
```

Comment:
```suggestion
             StringUtilities::ConvertCamelCaseToSnakeCase,
```

Reply:
that's unfortunately not my typo, but the given name of that function.

Reply:
Seems like a good opportunity to change it ;)

Reply:
I'd rather do it in a separate PR, otherwise this one will have a higher chance of getting blocked/ignored.